.github/workflows/post_check_svgs_comment.yml

```
@@ -36,13 +37,11 @@ jobs:
 
       - name: Comment on the PR about the result - Success
         uses: jungwinter/comment@v1 # let us comment on a specific PR
-        if: success() && steps.err_message_reader.outputs.content == 'None'
+        if: success() && steps.err_message_reader.outputs.content == '1'
```

I think we should use the enum here? 
```suggestion
        if: success() && steps.err_message_reader.outputs.content == SVG_STATUS_CODE.SVG_OK.value
```

---

I wanted to use the enum but the files are different. The enums are defined in a python file while the workflow is in YAML. I don't think there's a way to import it since they are different.